src/js/slider.js

```diff
@@ -0,0 +1,21 @@
+const charities = [
+  '01',
```

Comment:
Mamy podany cały obiekt z tymi instytucjami charytatywnymi. Dobrze by było go wkleić zamiast tego placeholdera. Bo będzie to też tablica obiektów wiec reszta kodu będzie też wyglądać trochę inaczej w oparciu o to